tests/TestRunner/app/ApiTests.js

```diff
@@ -432,4 +432,17 @@ describe(module.id, function () {
             expect(NSDate.dateWithTimeIntervalSinceReferenceDate(0).class()).toBe(NSDate);
         });
     }
+    
+    it("should be able to iterate over NSFastEnumeration objects", function () {
```

Comment:
I'll suggest trying with a bigger array (e.g. 30/32 elements), so the case where the JSIterator buffer is filled and moved to be covered.